test/test_helper.rb

```diff
@@ -116,7 +116,16 @@ def self.logger_adapter
 
   def self.persistence_adapter
     @persistence_adapter ||= begin
-                               db_config = ENV['DB_CONN_STRING'] || 'sqlite:/'
+                               db_config = ENV.fetch('DB_CONN_STRING') do
+                                 case ENV['DB']
+                                 when 'mysql'
+                                   "mysql2://root@127.0.0.1/#{ENV.fetch('MYSQL_DATABASE', 'travis_ci_test')}"
+                                 when 'postgresql'
+                                   "postgres://postgres@localhost/#{ENV.fetch('POSTGRES_DB', 'travis_ci_test')}"
```

Comment:
Not that it really matters, but travis is long gone
```suggestion
                                   "mysql2://root@127.0.0.1/#{ENV.fetch('MYSQL_DATABASE', 'ci_test')}"
                                 when 'postgresql'
                                   "postgres://postgres@localhost/#{ENV.fetch('POSTGRES_DB', 'ci_test')}"
```

Reply:
Nevermind, I'll do this in a followup